### PR TITLE
Pm 57 stretch goal ensure website responsiveness and change colors

### PR DESCRIPTION
**Pull request by dotte-osu**
Fixed the color and website responsiveness for all the pages except Favorite (it's not merged yet)

### CODE REVIEW

**Review by xHoudek — COMMENTED** [review]
The site looks good! The link in the login page is still blue. Not sure if you want that changed or not.

image

If we had more time in our project, I would suggest that we use `btn-primary` instead of `btn-info`, and just change the color of `btn-primary`. The reasoning is because it is our primary color, and not a color specific to information. Since we have like a week left to do everything, I think using `btn-info` is okay

**Comment by dotte-osu** [comment]
I didn't change the login page because it is just login. If you think we should change, I can change that!
I thought about changing the color for `btn-primary` but since it's bootstrap built in color, I thought it might be confusing if we change it. I see your point too. Let me know if I should change it in this ticket or next one.

**Comment by xHoudek** [comment]
I think it looks a little better when `text-info` is added to the link on the LoginPage.

Also no worries about changing `btn-info` back to `btn-primary`. I think that's something that would go on the back burner in a normal production environment, and it's not a big enough deal to change with our deadline coming up

**Comment by dotte-osu** [comment]
@xHoudek  Added text-info to the login page. Thank you!
image

File: src/routes/LoginPage/LoginPage.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default class LoginPage extends Component {
         <Section className='container'>
           <div className='loginWrapper'>
             <LoginForm shelter={this.state.shelter} />
-            <button className='btn btn-link' onClick={this.handlePageSwitch}>Don't have an account? Create one!</button>
+            <button className='btn btn-link text-info' onClick={this.handlePageSwitch}>Don't have an account? Create one!</button>
```
Comment (xHoudek) [inline src/routes/LoginPage/LoginPage.js:38]: Line 52 also needs the `text-info` class. After that, I think everything will be good to go!

Comment (dotte-osu) [reply src/routes/LoginPage/LoginPage.js:38]: oh no...! Thank you!

**Comment by dotte-osu** [comment]
Added text-info to line 52...!!! sorry!

**Review by xHoudek — APPROVED** [review]

**Pull request merged**